testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/security/RemoteIdentityTestCase.java

```diff
@@ -99,12 +99,4 @@ public void testSwitched() throws Exception {
         assertEquals("user1", targetBean.getPrincipalName("user1", "password1"));
     }
 
-    @Test
```

Comment:
@darranl I don't think this test has valid objective. EntryBean is unsecured stateless remote bean, so its principal is anonymous. This bean should not propagate guest identity to the injected SecurityInformation EJB in my opinion. The returned identity should be anonymous and not guest IMO. I do not think this test should be present for Elytron if the test objective is invalid.

Reply:
@Skyllarr I think it is correct that the injected bean is making use of the identity from the Remoting connection.

Reply:
@fjuma I see, after reading https://github.com/wildfly/wildfly/pull/12626#issuecomment-568093027 it probably makes sense to keep it here.

Reply:
@fjuma I see, after reading https://github.com/wildfly/wildfly/pull/12626#issuecomment-568093027 it probably makes sense to keep it here.